### DISCUSSION TITLE
com.example.samplestickerapp

### DISCUSSION THREAD

**Discussion by sahandileep** [WhatsApp/stickers#867] [Q&A] [answered]
hello friend, Could you help to fix this error

Build file 'C:\Users\sahan\Desktop\New folder (2)\stickers-master\stickers-master\Android\app\build.gradle' line: 73

Execution failed for task ':app:checkDebug'.

Your applicationId is currently com.example.samplestickerapp, please change your applicationId to a different string in app/build.gradle in line 10

**Comment by Zandor300** [comment, marked as answer]
Your applicationId is currently com.example.samplestickerapp, please change your applicationId to a different string in app/build.gradle in line 10
